indico/htdocs/sass/themes/administrative.scss

```diff
@@ -157,3 +169,21 @@ td.itemTopAlign{
 td.subItemOrder{
     font-family: Times;
 }
+
+.preformatted {
```

Comment:
wouldn't this make more sense in one of the global css files? having it in theme css files seems like a good way to miss it in some places